src/main/java/core/basesyntax/FileWork.java

```diff
@@ -8,7 +15,24 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
-    public String[] readFromFile(String fileName) {
-        return null;
+    public String[] readFromFile(String fileName) throws IOException {
```

Comment:
let's handle this exception with try catch

Reply:
1.added try - catch block.